system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -332,5 +344,9 @@ impl pallet_broker::Config for Runtime {
 	type WeightInfo = weights::pallet_broker::WeightInfo<Runtime>;
 	type PalletId = BrokerPalletId;
 	type AdminOrigin = EnsureRoot<AccountId>;
+	type SovereignAccountOf = SovereignAccountOf;
+	// Roughly 1.5 years with a period of 28 days(~ish)
+	// TODO: @szegoo, can you help checking this value?
+	type MaxAutoRenewals = ConstU32<20>;
```

Comment:
Kusama has 100 cores, so I think we could also set this to 100, unless there are short-term plans to increase the core count. Even if the core count increases in the near future, I doubt all of them will have auto-renewal enabled.
```suggestion
	type MaxAutoRenewals = ConstU32<100>;
```